assets/js/util.js

```diff
@@ -293,13 +305,62 @@ function callApy(options, endpoint) {
     var requestUrl = window.location.protocol + window.location.hostname +
         window.location.pathname + '?' + $.param(requestOptions.data);
 
+    if(!installationNotificationShown) {
```

Comment:
Yes, because the notification is not yet shown! Are you suggesting that the value holds even when the user refreshes?